tests/python/relay/test_op_level10.py

```diff
@@ -380,10 +382,31 @@ def verify_dynamic_batch_matmul(x_shape, y_shape, out_shape, dtype="float32"):
 # TODO(mbrookhart): enable once VM supports heterogenous execution
 # @tvm.testing.uses_gpu
```

Comment:
I thought I'd removed all of these TODO's. Oops! This should work on GPU now, I think

Reply:
I'm not sure, I think [this line](https://github.com/apache/tvm/blob/84359a97bdb896ddd2a74c65631a00da1ec5ec70/python/tvm/topi/nn/batch_matmul.py#L65-L65) (which is called by topi.cuda [here](https://github.com/apache/tvm/blob/90dce48de6ca74a439ed24fc3a3d80d57b305dd0/python/tvm/topi/cuda/batch_matmul.py#L32-L32)) will fail if `x.shape[2]` is static and `y.shape[2]` is dynamic.

By the way, the unit test in ci reported a strange error. I’m not sure if the error is related to this PR.

Reply:
This is a flaky test that times out sometimes, rebase?

Reply:
Everything is fine now, thanks.